src/main/java/dataloader/utils/ParseUtils.java

```diff
@@ -0,0 +1,16 @@
+package dataloader.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseUtils {
+
+    public static List< Double > parseAsDouble(List<Object> dataSet){
+
+        List<Double> data = new ArrayList<Double>( dataSet.size());
+        for(int i=0; i< dataSet.size(); ++i){
```

Comment:
what do you think about iterations with [forEach](https://www.baeldung.com/foreach-java), [map](https://www.mkyong.com/java8/java-8-streams-map-examples/) or [enhanced for loops](https://www.programiz.com/java-programming/enhanced-for-loop)? could one of these replace this for loop?

Reply:
yes probably you are right should go with ```forEach```

Reply:
agree.